sdks/python/setup.py

```diff
@@ -89,7 +89,7 @@ def get_version():
     'google-apitools>=0.5.2',
     'httplib2>=0.8',
     'mock>=1.0.1',
-    'oauth2client>=2.0.1,<4.0.0',
+    'oauth2client>=2.0.1,<4.0.0dev',
```

Comment:
This is a pretty uncommon edge case, but judging by https://www.python.org/dev/peps/pep-0440/#handling-of-pre-releases, it could potentially happen if the user had already installed oauth2client 4.0.0dev, right?